### PR TITLE
Enable touch events

### PR DESCRIPTION
**Pull request by fernandomatal**
Enable touch on indicators and delegate methods to handle the action.

### CODE REVIEW

**Comment by fernandomatal** [comment]
Hey, I updated my fork to be on track with your repo. 

Just in case you are still thinking on accepting this or not.

**Comment by realityworks** [comment]
@fernandomatal have you heard anything from the Chilli team?

**Comment by fernandomatal** [comment]
Hey @realityworks, not really. I asked about this long time ago and they told me to do a PR with the change, I did it but they never replied. They've been barely active since, but no news from them.

File: CHIPageControl/CHIPageControlAji.swift [hunk 1]
```diff
@@ -105,4 +105,17 @@ open class CHIPageControlAji: CHIBasePageControl {
         return CGSize(width: CGFloat(inactive.count) * self.diameter + CGFloat(inactive.count - 1) * self.padding,
                       height: self.diameter)
     }
+    
+    override open func didTouch(gesture: UITapGestureRecognizer) {
+        var touchIndex: Int?
+        let point = gesture.location(ofTouch: 0, in: self)
+        inactive.enumerated().forEach({ count, layer in
```
Comment (chili-ios) [inline CHIPageControl/CHIPageControlAji.swift]: I think this way would be more compact and it avoids force casting. 
```
if let touchIndex = inactive.enumerated().first(where: { $0.element.hitTest(point) != nil })?.offset {
    delegate?.didTouch(pager: self, index: touchIndex)
}
```

Can be changed for all other page controls as well.

Comment (fernandomatal) [reply CHIPageControl/CHIPageControlAji.swift]: Great, I just updated my repo with this change.

**Pull request merged**